content/guides/docker-concepts/building-images/build-tag-and-publish-an-image.md

```diff
@@ -0,0 +1,228 @@
+
+### Building images
+
+Most often, images are built using a `Dockerfile`. The most basic `docker build` command might look like the following:
+
+```bash
+docker build .
+```
+
+The final `.` in the command provides the path or URL to the [build context](https://docs.docker.com/build/building/context/#what-is-a-build-context). At this location, the builder will find the `Dockerfile` and other referenced files.
+
+When a build occurs, the builder will pull the base image if needed and then run all of the instructions as outlined in the `Dockerfile`.
+
+With the previous command, the image will have no name, but the output will provide the ID of the image. As an example, the previous command might produce the following output:
+
+```console
+$ docker build .
+[+] Building 3.5s (11/11) FINISHED                                              docker:desktop-linux
+ => [internal] load build definition from Dockerfile                                            0.0s
+ => => transferring dockerfile: 308B                                                            0.0s
+ => [internal] load metadata for docker.io/library/python:3.12                                  0.0s
+ => [internal] load .dockerignore                                                               0.0s
+ => => transferring context: 2B                                                                 0.0s
+ => [1/6] FROM docker.io/library/python:3.12                                                    0.0s
+ => [internal] load build context                                                               0.0s
+ => => transferring context: 123B                                                               0.0s
+ => [2/6] WORKDIR /usr/local/app                                                                0.0s
+ => [3/6] RUN useradd app                                                                       0.1s
+ => [4/6] COPY ./requirements.txt ./requirements.txt                                            0.0s
+ => [5/6] RUN pip install --no-cache-dir --upgrade -r requirements.txt                          3.2s
+ => [6/6] COPY ./app ./app                                                                      0.0s
+ => exporting to image                                                                          0.1s
+ => => exporting layers                                                                         0.1s
+ => => writing image sha256:9924dfd9350407b3df01d1a0e1033b1e543523ce7d5d5e2c83a724480ebe8f00    0.0s
+```
+
+With the previous output, you could start a container by using the referenced image:
+
+```console
+docker run sha256:9924dfd9350407b3df01d1a0e1033b1e543523ce7d5d5e2c83a724480ebe8f00
+```
+
+That name certainly isn't memorable, which is where tagging becomes useful.
+
+
+### Tagging images
+
+Tagging images is the method to provide an image with a memorable name. However, there is a structure to the name of an image. A full image name has the following structure:
+
+```
```

Comment:
I haven't done a second look through yet. This is one I missed that got caught by the linter.

```suggestion
```text
```